.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker
+
+on:
+  push:
+    # Publish `master` as Docker `latest` image.
+    branches:
+      - master
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
```

Comment:
We haven't really been using tags at this point- think we should start? Might be helpful I guess to have it available for certainty of which version is currently running in prod, but I've just been keeping prod synced with master's latest commit

Reply:
The main reason it is here is using tags for branches (and the corresponding container images) is it makes rolling back possible. May not be crucial now but could be later.

Reply:
I know what tags *do*. I am asking for a discussion on whether it is valuable for us to move towards a tag-based approach when all pushes to master are already being deployed as they happen, albeit manually at this time.

Rolling back is already possible through this method so the tags do not currently add anything of value developer-wise. It might, however, be useful for documentation, which is why I asked before. Do you personally think it is useful for us to move to a tag-based approach at this time for documentation purposes, mirroring the deployments with the Releases tab on Github?